releasing/versioning_pre_nightly.md

```diff
@@ -0,0 +1,70 @@
+## Versioning in nightly and prerelease binaries
+
+Binary packages produced for prerelease and nightly builds have some
+particularities to establish the priority among them nicely.
+
+**Prereleases** for version `X.Y.Z` are created as candidates of the stable
+release of `X.Y.Z` so the stable needs to have precedence over prereleases when
+both are available through different repositories.
+
+**Nightlies** for version `X.Y.Z` are snapshots of development code merged on
+top of a released version `X.Y.Z` features and/or patches that will be shipped
+into the next release. Therefore nightlies needs to have precedence over stable
+releases. Another use case for nightlies is to server as continuous unstable
+releases when preparing a new version `X.Y.Z`. In this scenario the stable
```

Comment:
```suggestion
releases. Another use case for nightlies is to serve as continuous unstable
releases when preparing a new version `X.Y.Z`. In this scenario the stable
```

I'm not sure what this statement means?

Reply:
Imagine that you are going to release version 2.0.0 and the current software is in version 1.9.3. Before the 2.0,0 release there are PRs merged everyday in the `main` branch and the development is still open. At that point you also want to have packages for testing generated every day with the changes in ` main` branch. The nightlies will have the version 1.99.99 and will serve as small releases to test `main` branch changes. Does it make sense to you now?